common/build.gradle

```diff
@@ -25,4 +25,5 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.3.1'
     testCompile 'junit:junit:4.12'
+    compile "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
```

Comment:
TODO: Should use a hard-wired version number here independent of the root project.

Reply:
Will send a commit to resolve